src/workspace-hack/Cargo.toml

```
@@ -79,6 +79,7 @@ reqwest = { version = "0.11", features = ["__tls", "default-tls", "hyper-tls", "
 ring = { version = "0.16", features = ["alloc", "dev_urandom_fallback", "once_cell", "std"] }
 scopeguard = { version = "1", features = ["use_std"] }
 serde = { version = "1", features = ["alloc", "derive", "rc", "serde_derive", "std"] }
+simd-json = { version = "0.7", features = ["allow-non-simd", "serde", "serde_impl", "serde_json", "swar-number-parsing"] }
```

```suggestion
simd-json = { version = "0.7", features = ["serde", "serde_impl", "serde_json", "swar-number-parsing"] }
```

I prefer to explicitly reject the non-supported platforms, because we don't have any tests on them.

---

Enable `allow-non-simd` will disable all simd optimizations of `simd-json` and only rely on LLVM, I agree with @fuyufjh .

---

looks like a bug of workspace hack here. I come up with no solution. @wangrunji0408

---

Don't know how workspace-hack works. But I tried to remove `simd-json-derive = "0.7"`  and then `simd-json` disappeared from workspace-hack.